File: go/callee-service/callee/calleelogic.go
package callee

func SayMyName(name string) Callee {
	return Callee{ID: "0", Message: "Your name is: " + name}
}

func SayMyOtherName(name string) Callee {
	return Callee{ID: "0", Message: "Your other name is: " + name}
}

func Save(callee Callee) Callee {
	return Callee{ID: "0", Message: "Storing your message: " + callee.Message}
}

File: python/callee-service-python/src/callee_service/logic/callee_logic.py
from callee_service.controller.dto.callee import Callee


class CalleeLogic:
    async def say_my_name(self, name: str | None) -> Callee:
        return Callee(id="0", message=f"Your name is: {name}")

    async def say_my_other_name(self, name: str | None) -> Callee:
        return Callee(id="0", message=f"Your other name is: {name}")

    async def save(self, callee: Callee) -> Callee:
        return Callee(id="0", message=f"Storing your message: {callee.message}")
